onnxruntime/core/session/inference_session.cc

```diff
@@ -868,7 +868,7 @@ common::Status InferenceSession::SaveToOrtFormat(const PathString& filepath) con
     uint8_t* buf = builder.GetBufferPointer();
     int size = builder.GetSize();
     file.write(reinterpret_cast<const char*>(buf), size);
-    ORT_RETURN_IF_NOT(file, "Failed to save ORT format model to file: ", ToUTF8String(filepath));
+    ORT_RETURN_IF_NOT(file, "Failed to save ORT format model to file: ", ToUTF8String(filepath.native()));
```

Comment:
can path::string() be used here?

Reply:
No. I believe filepath.string() will return a string in native encoding, which is something like "ISO-8859-1" or "GBK" or something else. But, here we expect a UTF-8 string.